.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = clang
 	url = https://github.com/RadeonOpenCompute/hcc-clang-upgrade.git
 	branch = clang_tot_upgrade
+[submodule "compiler-rt"]
```

Comment:
You are adding a new submodule here, but the Jenkinsfile I recently checked in checks out the submodules manually.  If you want CI to also build compiler-rt, you need to add a new [git clone into Jenksinsfile](https://github.com/scchan/hcc/blob/add_compiler-rt/Jenkinsfile#L32).

I know this is sub-optimal, but cloning with `git clone --recursive -b <name>` takes greater than 15 minutes per build.  Sometimes, this clone step would time out and fail the build, also sub-optimal.  As a compromise, I manually clone each submodule with depth 1 which makes the whole clone operation take < 2 minutes.  The downside is two files to update when new submodules are added/deleted (hopefully infrequently).